storage/objects/stream_file_upload.go

```diff
@@ -43,8 +43,24 @@ func streamFileUpload(w io.Writer, bucket, object string) error {
 	ctx, cancel := context.WithTimeout(ctx, time.Second*50)
 	defer cancel()
 
+	o := client.Bucket(bucket).Object(object)
+
+	// Set a generation-match precondition. The request to upload is aborted
+	// if the object's generation number does not match your precondition
+	// criteria. This avoids race conditions and data corruption.
+	// For an object that does not yet exist, set the DoesNotExist precondition.
+	o = o.If(storage.Conditions{DoesNotExist: true})
+
+	// If the live object already exists in your bucket, instead set a generation
+	// match condition using the live object's generation number as follows.
```

Comment:
"as follows" is not needed (makes the comment here consistent with the one above)